### DISCUSSION TITLE
How to create half donut?

### DISCUSSION THREAD

**Discussion by cjmling** [Abhinandan-Kushwaha/react-native-gifted-charts#578] [General]
I tried to look into doc for example, but could not find. Not sure if this is possible?

![image](https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/assets/1612187/f2c8d686-f886-424a-a504-5a34f3575229)

**Comment by Abhinandan-Kushwaha** [comment]
Use the `semiCircle` prop.

**Comment by Abhinandan-Kushwaha** [comment]
Use the `semiCircle` prop.
